packages/manager/modules/storages/src/netapp/netapp.service.js

```diff
@@ -0,0 +1,19 @@
+export default class NetappService {
```

Comment:
Does this service is still a draft?

- `$q` injected but not used.
- Empty `.then()` block.